### PR TITLE
RavenDB-18226 - Sharding - Connection Strings View Endpoints

### PR DESCRIPTION
**Pull request by shiranshalom**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-18226

### Additional description

implemented **GET** `/databases/*/admin/periodic-backup/config` EP

### Type of change

- New feature

### How risky is the change?

- Not relevant

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

**Comment by Danielle9897** [comment]
@shiranshalom we need also
GET `/databases/*/admin/backup-data-directory`
(used for the local destination)

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs [hunk 1]
```diff
@@ -291,46 +291,51 @@ public class OfflineMigrationValidation
         [RavenAction("/studio-tasks/periodic-backup/test-credentials", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
         public async Task TestPeriodicBackupCredentials()
         {
-            var type = GetQueryStringValueAndAssertIfSingleAndNotEmpty("type");
+            await TestPeriodicBackupCredentials(this);
+        }
+
+        public static async Task TestPeriodicBackupCredentials(RequestHandler requestHandler)
```
Comment (ppekrol) [inline src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: why not wrap it up in a single processor?

File: src/Raven.Server/Web/System/OngoingTasksHandler.cs [hunk 2]
```diff
@@ -307,19 +307,24 @@ private IEnumerable<OngoingTask> CollectExternalReplicationTasks(List<ExternalRe
 
         [RavenAction("/databases/*/admin/periodic-backup/config", "GET", AuthorizationStatus.DatabaseAdmin)]
         public async Task GetConfiguration()
+        {
+            await GetBackupConfiguration(this);
+        }
+
+        public static async Task GetBackupConfiguration(RequestHandler requestHandler)
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/OngoingTasksHandler.cs]: why not wrap it up in a single processor?

**Review by ppekrol — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/System/OngoingTasksHandler.cs (hunk 2 above)
Comment (shiranshalom) [reply src/Raven.Server/Web/System/OngoingTasksHandler.cs]: done

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs (hunk 1 above)
Comment (shiranshalom) [reply src/Raven.Server/Web/Studio/StudioTasksHandler.cs]: done

**Pull request merged**